cirq/contrib/two_qubit_gates/example.py

```diff
@@ -0,0 +1,61 @@
+from time import time
+
+import numpy as np
+from matplotlib import pyplot as plt
+
+# pylint: disable=line-too-long
+from cirq import FSimGate, unitary
+from cirq.contrib.two_qubit_gates.gate_compilation import gate_product_tabulation
+from cirq.contrib.two_qubit_gates.math_utils import random_two_qubit_unitaries_and_kak_vecs, unitary_entanglement_fidelity
+
+# pylint: enable=line-too-long
+
+
+def main():
+    # coverage: ignore
```

Comment:
Optional: MIght be a good idea to have a test that just runs through this and makes sure it doesn't err. Otherwise, this could stop working and we wouldn't know. I think if you do that, you can probably remove these coverage: ignore pragmas.

Reply:
Example code should still be covered. It's okay if the test just runs the method and confirms it doesn't throw an error, without checking what it actually did.

Reply:
Done